src/com/intellij/idea/plugin/hybris/tools/ccv2/actions/CCv2BuildActions.kt

```diff
@@ -108,12 +118,49 @@ class CCv2DeleteBuildAction(
 class CCv2FetchBuildsAction : AbstractCCv2FetchAction<CCv2BuildDto>(
     tab = CCv2Tab.BUILDS,
     text = "Fetch Builds",
-    icon = HybrisIcons.CCv2.Actions.FETCH,
+    icon = FETCH,
     fetch = { project, subscriptions, onStartCallback, onCompleteCallback ->
         CCv2Service.getInstance(project).fetchBuilds(subscriptions, onStartCallback, onCompleteCallback)
     }
 )
 
+class CCv2FetchBuildDetailsAction(
+    private val subscription: CCv2Subscription,
+    private val build: CCv2BuildDto,
+    private val onStartCallback: () -> Unit,
+    private val onCompleteCallback: (CCv2BuildDto) -> Unit
+) : DumbAwareAction("Fetch Build", null, FETCH) {
+
+    private var fetching = false
+
+    override fun getActionUpdateThread(): ActionUpdateThread = ActionUpdateThread.BGT
+
+    override fun actionPerformed(e: AnActionEvent) {
+        val project = e.project ?: return
+        CCv2Service.getInstance(project).fetchBuildWithCode(
+            subscription, build.code,
+            {
+                fetching = true
+                e.presentation.text = "Fetching..."
+
+                onStartCallback.invoke()
+            },
+            { response ->
+                invokeLater {
+                    fetching = false
+                    e.presentation.text = "Fetch Build"
+
+                    onCompleteCallback.invoke(response)
+                }
+            }
+        )
+    }
+
+    override fun update(e: AnActionEvent) {
+        e.presentation.isEnabled = !fetching && ApplicationSettingsComponent.getInstance().state.ccv2Subscriptions.isNotEmpty()
```

Comment:
could you please add similar to this check?
```
        val isRightPlace = "GoToAction" != e.place
        e.presentation.isEnabled = isRightPlace && isEnabled()
        e.presentation.isVisible = isRightPlace && e.project
            ?.let { CCv2View.getActiveTab(it) == tab }
            ?: false
```

I'll update other similar actions, we do not want to show our actions in the `Actions GoTo` search.

Reply:
Resolved.